BiweeklyMeetings/2022-06-28.md

```diff
@@ -0,0 +1,442 @@
+</sub>  
+Editors meetings are [public](https://discord.gg/36ezQYTvgk), [recorded](https://www.youtube.com/c/CardanoFoundation/videos) and [transcript](https://github.com/cardano-foundation/CIPs/tree/master/BiweeklyMeetings): do join and participate for discussions/PRs of significances to you.  
+
+
+## Editors Meeting Flow
+
+**Triage/Review**: Some CIPs might fall out of grace or not get updated, a CIP that hasn’t seen activity for 3 months should be checked on, and appropriate action taken. Ex: did any of the recent changes obsolete current CIPs? Consider ‘Active’ -> ‘Obsolete’ transitions..  
+
+**Last Check**: Review of the PRIOR meetings Decisions  - if no objection, apply change (effectively a two week lag from decision to action, as a grace period)  
+
+**New CIPs Review**: CIPs up for review should be looked over collectively, with discussion where needed. (on top of the asynchronous reviews)  
+
+PR -> ‘Draft’: Needs format + approval.  
+‘Draft’ -> ‘Proposed’: Needs a PLAN towards Active + implementation.  
+‘Proposed’ -> ‘Active’:  Objective criteria as laid out observed, and consensus agreeing.   
+**Current Discussions**: What the current CIPs discussions are on social media / forums / Discord.  
+
+**Issues**: Review of the open Issues on the CIP GitHub 
+
+**Close**: Recap of actions taken and decisions. List the CIPs that are due for review.  
+
+
+## June 28 2022 Transcript
+**Attending Editors**: Matthias Benkort, Sebastien Guillemot, Frederic Johnson, Robert Phair. 
+**Guests**: Morgan Thomas, Michael Liesenfelt
+
+### Triage
+
+#### PR271
+[PR271 CIP-0027? | Update Proposal: Multiple Royalty Addresses](https://github.com/cardano-foundation/CIPs/pull/271)
+
+**Matthias** -: CIP-0027? | Update Proposal: Multiple Royalty Addresses
+  And that current CIP is already active. It's been used by a few marketplaces. So it doesn't quite make sense to modify it as much. As for the proposal itself, I made already some remark last week. I think the author came back saying yes, true. But that would make for a worse UX. And therefore I would prefer the approach. They have a client in the proposal, which is fair enough, but I still maintain that it should be made as a separate proposal, which is an extension of the existing one rather than modifying what exists and has been used in a while.
+
+**M. Àngels** -:  So I see you maintain a conversation. So from my understanding, Nicolas will need to make a separate, right?
+
+**Matthias** -: Yes, indeed.
+
+**M. Àngels** -:  So shall we add that we are waiting for the author?
+
+**Matthias** -: I think that's pretty much the case now because the proposal by itself...
+
+**Robert** -: [Issues with the audio] I don't think based on what Matthias was saying before, which he'd probably be staying now, if he could. It sounds like that can only really go ahead as a separate CIP because of the existing implementations that it would be incompatible with.
```

Comment:
```suggestion
**Robert** -: [Issues with the audio: Robert cannot hear Matthias] I don't think based on what Matthias was saying before, which he'd probably be staying now, if he could. It sounds like that can only really go ahead as a separate CIP because of the existing implementations that it would be incompatible with.
```
I hope adding some context to the annotations will clarify the otherwise confusing dialogue.  Again I apologise for the issue on my end (I could hear Maria but not Matthias).  FYI Discord says "occasionally" some users cannot hear others & the fix is to refresh the web app (Ctrl-R) but it was about 10 minutes before I could figure this out. 😖

Reply:
That was an interesting moment :sweat_smile: